.github/workflows/integration-tests.yml

```
@@ -51,23 +51,26 @@ jobs:
         shell: bash
       - name: Check conversion coverage
         run: |
-          export RUNNER_OS=${{ runner.os }}
           ./tests/coverage-test.sh
         shell: bash
       - name: Run Python doc tests
         run: |
           python -m doctest qdrant_client/local/local_collection.py
       - name: Run integration tests
+        env:
+          HF_TOKEN={{ secrets.HF_TOKEN }}
```

actionlint (1.7.4)

expecting a single ${{...}} expression or mapping value for "env" section, but found plain text node

(syntax-check)